### PR TITLE
Create Simplified Chinese Version of README.rst

### PR DESCRIPTION
**Pull request by LShun**
### Description

Hello, I translated the README.rst to Simplified Chinese using Google Translate and my own understanding in Simplified Chinese language. I hope this can help people who are not fluent in English but are fluent in Chinese to be able to understand the project easier and faster. There are some words that I didn't convert to Chinese either because they are special names, or specific links which does not have a translated version ( for example at line 68: LICENSE_ an NOTICE_ ). The file name is README-zh.rst, but I don't mind if its changed, its just for easier initial identification.
### Status

done, ready for review
### Checklist (tick everything that applies)
- [ ] [Code linting](http://libcloud.readthedocs.org/en/latest/development.html#code-style-guide) (required, can be done after the PR checks)
- [v] Documentation
- [ ] [Tests](http://libcloud.readthedocs.org/en/latest/testing.html)
- [ ] [ICLA](http://libcloud.readthedocs.org/en/latest/development.html#contributing-bigger-changes) (required for bigger changes)

### CODE REVIEW

**Comment by tonybaloney** [comment]
I'm going to struggle to code review this one! I can't read chinese. Open to anyone for review

**Comment by watermelo** [comment]
I think the translation is not good enough to understand and exist some typos.

**Comment by LShun** [comment]
I see, thanks. Would you mind to specify the typo's and the parts which are
hard to understand so I can correct them?

File: README-zh.rst [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+.. image:: https://img.shields.io/irc/%23libcloud.png
+        :target: http://webchat.freenode.net/?channels=libcloud
+
+.. image:: https://bestpractices.coreinfrastructure.org/projects/152/badge
+        :target: https://bestpractices.coreinfrastructure.org/projects/152
+
+.. image:: https://coveralls.io/repos/github/apache/libcloud/badge.svg?branch=trunk
+        :target: https://coveralls.io/github/apache/libcloud?branch=trunk
+
+Apache Libcloud 是一个Python 的库，隐藏不同云端供着的应用程序接口（API's) 和允许您通过统一和易于使用的API和 Python library管理不同的云端资源。
```
Comment (watermelo) [inline README-zh.rst]: Apache Libcloud是一个Python库，它隐蔽了不同云资源API层的差异，并允许您通过统一且易于使用的API来管理不同的云资源。

File: README-zh.rst [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+* **计算** - 云服务器和块存储 - 服务例子： Amazon EC2 and Rackspace
+  Cloud Servers (``libcloud.compute.*``)
+* **存储** - 云对象存储 and 内容交付网络(CDN-Content Delivery Network ）  - 服务例子： Amazon S3 and Rackspace
+  CloudFiles (``libcloud.storage.*``)
+* **负载平衡器** - 作为服务的负载平衡器, LBaaS (``libcloud.loadbalancer.*``)
+* **域名服务器 ( DNS - Domain name server )** - 作为服务的域名服务器, DNSaaS (``libcloud.dns.*``)
+* **容器** - 虚拟化服务容器 (``libcloud.container.*``)
+
+
+Apache Libcloud 是一个 Apache 的计划, 往 <http://libcloud.apache.org> 看以获得跟多资料
```
Comment (watermelo) [inline README-zh.rst]: 更多资料，不是跟多

File: README-zh.rst [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+贡献
+==
+
+有关如何贡献的信息，请参阅贡献在我们的文档中的贡献章节 <https://libcloud.readthedocs.org/en/latest/development.html#contributing>
+
+执照
+==
+
+Apache Libcloud是根据Apache 2.0被许可证授权的。 更多有关详情，请参阅 LICENSE_ 和 NOTICE_ 文件。
```
Comment (watermelo) [inline README-zh.rst]: Apache Libcloud是被Apache 2.0许可证授权的。

File: README-zh.rst [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+.. image:: https://coveralls.io/repos/github/apache/libcloud/badge.svg?branch=trunk
+        :target: https://coveralls.io/github/apache/libcloud?branch=trunk
+
+Apache Libcloud 是一个Python 的库，隐藏不同云端供着的应用程序接口（API's) 和允许您通过统一和易于使用的API和 Python library管理不同的云端资源。
+
+您可以使用Libcloud管理的资源分为以下类别：
+
+* **计算** - 云服务器和块存储 - 服务例子： Amazon EC2 and Rackspace
+  Cloud Servers (``libcloud.compute.*``)
+* **存储** - 云对象存储 and 内容交付网络(CDN-Content Delivery Network ）  - 服务例子： Amazon S3 and Rackspace
```
Comment (watermelo) [inline README-zh.rst]: and改成和

**Review by watermelo — COMMENTED** [review]
hi~ @LShun, Correction is as follows, hope be modified.

File: README-zh.rst (hunk 3 above)
Comment (LShun) [reply README-zh.rst]: Thank you, I edited it with your correction.

File: README-zh.rst (hunk 2 above)
Comment (LShun) [reply README-zh.rst]: Thank you, I edited it with your suggestion.

File: README-zh.rst (hunk 1 above)
Comment (LShun) [reply README-zh.rst]: Thank you, I edited it with your correction.

File: README-zh.rst (hunk 4 above)
Comment (LShun) [reply README-zh.rst]: Thank you, I edited it with your correction.

**Pull request closed without merging**